pkg/storage/trb_request_attendee.go

```diff
@@ -0,0 +1,104 @@
+package storage
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	"github.com/google/uuid"
+
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// CreateTRBRequestAttendee creates a new TRB request attendee record in the database
+func (s *Store) CreateTRBRequestAttendee(ctx context.Context, attendee *models.TRBRequestAttendee) (*models.TRBRequestAttendee, error) {
+	attendee.ID = uuid.New()
+	const createTRBRequestAttendeeSQL = `
+		INSERT INTO trb_request_attendees (
+			id,
+			eua_user_id,
+			trb_request_id,
+			role,
+			component,
+			created_by,
+			modified_by
+		)
+		VALUES (
+			:id,
+			:eua_user_id,
+			:trb_request_id,
+			:role,
+			:component,
+			:created_by,
+			:modified_by
+		)`
+	_, err := s.db.NamedExecContext(
+		ctx,
+		createTRBRequestAttendeeSQL,
+		attendee,
+	)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error("Failed to create TRB request attendee with error %s", zap.Error(err))
+		return nil, err
+	}
+	return attendee, nil
+}
+
+// UpdateTRBRequestAttendee updates a TRB request attendee record in the database
+func (s *Store) UpdateTRBRequestAttendee(ctx context.Context, attendee *models.TRBRequestAttendee) (*models.TRBRequestAttendee, error) {
+	const sql = `
+		UPDATE trb_request_attendees
+		SET
+			role = :role,
+			component = :component,
+			modified_by = :modified_by,
+			modified_at = CURRENT_TIMESTAMP
+		WHERE trb_request_attendees.id = :id
+	`
+	_, err := s.db.NamedExecContext(ctx, sql, attendee)
```

Comment:
Slight issue here, when you update the TRB attendee, it doesn't the values from the DB, notably euaUserID, or createdAt, createdBy.

Could address a couple of ways, I add a returning clause to the end of the sql to just return all the values from the db.

![image](https://user-images.githubusercontent.com/99294987/191806100-e80d496c-2911-4ef5-8b22-506119865366.png)

![image](https://user-images.githubusercontent.com/99294987/191806138-25dd9f05-d846-4d5f-a1ec-0c6b247abd21.png)

Reply:
I removed those from being edible per @ClayBenson94  request in a comment on this PR. Need to confirm which way it should be.

Reply:
I think those not being editable is good, but I think the other values should probably be returned from the DB as well as the edited ones.

Reply:
@jstrang-oddball Can you clarify which comment? I think I either misspoke or perhaps there was a misunderstanding.

It seems like what @StevenWadeOddball is requesting is not that these fields be editable, but that when you perform the update statement, the newly set values in the DB are returned in the response, rather than having to perform another query to fetch those values.